src/extension/inlineEdits/vscode-node/jointInlineCompletionProvider.ts

```diff
@@ -181,7 +181,10 @@ export class JointCompletionsProviderContribution extends Disposable implements
 
 					// @ulugbekna: note that we don't want it if modelUnification is on
 					const modelUnification = unificationStateValue?.modelUnification ?? false;
-					if (!modelUnification || unificationStateValue?.codeUnification || extensionUnification || configEnabled || this._copilotToken.read(reader)?.isNoAuthUser) {
+					if (
+						(!modelUnification || unificationStateValue?.codeUnification || extensionUnification || configEnabled || this._copilotToken.read(reader)?.isNoAuthUser) &&
+						!excludes.includes('completions')
```

Comment:
To maintain the existing logic it should be `(!this.inlineEditsEnabled.read(reader) || !excludes.includes('completions'))`